docs/avdanos/contributing/design/OXYDE/intro.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: OXYDE
+description: Welcome to OXYDE.
+---
+
+:::danger
+This version is in Beta, which means is subject to updates and changes in the following days until the Beta phase is completed.
+:::

```

Comment:
This is note is inconsistent with the others — the others are `caution`. Which one is it?